cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ModelUtils.java

```diff
@@ -739,4 +739,15 @@ public static boolean hasOwnerReference(HasMetadata resource, OwnerReference own
     public static int caCertGeneration(Ca ca) {
         return Annotations.intAnnotation(ca.caCertSecret(), Ca.ANNO_STRIMZI_IO_CA_CERT_GENERATION, Ca.INIT_GENERATION);
     }
+
+    /**
+     * Returns the id of a pod given the pod name
+     *
+     * @param podName   Name of pod
+     *
+     * @return          Id of the pod
+     */
+    public static int idOfPod(String podName)  {
+        return Integer.parseInt(podName.substring(podName.lastIndexOf("-") + 1));
```

Comment:
Seem to remember that the PodSets proposal suggested that the brokers pods could use any id scheme (a, b, c or 1st, 2nd, 3rd)? Is there any chance these ids won't be an int? Does it matter if they stay a string?

Reply:
The ultimate Kafka broker ID has to be int AFAIK. So there will be a way how to convert it to int.

Reply:
Thanks Jakub, thought that might be the case, I was just being pedantic.